src/classes/CSVBatchReader.cls

```diff
@@ -0,0 +1,79 @@
+/**
+* File Name :   CSVBatchReader.cls
+* Description : Class reding from CSV file in batches
+*               Modified version of http://developer.financialforce.com/customizations/importing-large-csv-files-via-batch-apex/
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 02/04/2014 Klea Kolaric   Initial version
+* 0.2 08/08/2014 Cristina P		Upload Audience Group and Members in the same time
+*
+*
+*/
+public with sharing class CSVBatchReader implements Database.batchable<String>, Database.Stateful {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.